publify_core/app/assets/javascripts/publify_admin.js

```diff
@@ -76,7 +76,7 @@ function tag_manager() {
 }
 
 function save_article_tags() {
-  $('#article_keywords').val($('#article_form').find('input[name="hidden-article[keywords]"]'));
+  $('#article_keywords').val($('#article_form').find('input[name="hidden-article[keywords]"]')["0"].defaultValue);
```

Comment:
This seems dangerous for two reasons:
(1) Why are you indexing into an array using a string?
(2) Why are you using the default value from that input field, rather than the actual value that a user has typed in?